docs/docs/install/podman-quadlet.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 90
+---
+
+# Podman deploy with quadlets
+
+You can deploy Immich on Podman using quadlets.
+
+Here are some sample rootless quadlet container files that can be placed in /etc/containers/systemd/users/${ID} where ID is the uid of whatever your rootless user is.
+
+Please note you'll need :z or :Z for selinux enabled hosts.
+
+immich-database.container
+```bash
+[Unit]
+Description=Immich Database
+Requires=immich-redis.service
+
+[Container]
+AutoUpdate=registry
+EnvironmentFile=${location_of_env_file}
+Image=registry.hub.docker.com/tensorchord/pgvecto-rs:pg16-v0.2.1
+Label=registry
+Network=slirp4netns:port_handler=slirp4netns
+PublishPort=5432:5432
+Volume=${host_database_directory}:/var/lib/postgresql/data:z
+Volume=/etc/localtime:/etc/localtime:ro
+
+[Service]
+Restart=always
+
+[Install]
+WantedBy=multi-user.target default.target
+```
+
+immich-microservices.container
+```bash
+[Unit]
+Description=Immich Microservices
+Requires=immich-redis.service immich-database.service
+
+[Container]
+#AddDevice=/dev/dri:/dev/dri #Needed for HWA
+#AddDevice=nvidia.com/gpu=0 #Needed for nvidia HWA, after setting up container tools
```

Comment:
May want to consider [upstream documentation](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/cdi-support.html) which I believe suggests that for podman use of CDI is suggested?

Reply:
If you are using CUDA you use AddDevice=nvidia.com/gpu=0 after setting up CDI, and if you're using VAAPI you would do AddDevice=/dev/dri:/dev/dri. You can find the information in the link you posted under: Running a Workload with CDI.